pkg/acsengine/template_generator.go

```diff
@@ -975,6 +975,30 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) templat
 					} else {
 						val = cloudSpecConfig.KubernetesSpecConfig.NVIDIAImageBase + KubeConfigs[k8sVersion][NVIDIADevicePluginAddonName]
 					}
+				case "kubernetesNVIDIADevicePluginCPURequests":
+					if aC > -1 {
```

Comment:
I think this should be nC, instead of aC.

nC := getAddonContainersIndexByName(nvidiaDevicePluginAddon.Containers, NVIDIADevicePluginAddonName)

Reply:
indeed, will file a quick fix, good find!